atlassian/jira.py

```diff
@@ -590,7 +590,7 @@ def remove_group(self, name, swap_group=None):
         return self.delete(url, params=params)
 
     def get_all_users_from_group(
-            self, group, include_inactive_users=False, start=0, limit=50
+        self, group, include_inactive_users=False, start=0, limit=50
```

Comment:
That's the result of black which was added to tox by @mandrean 9 days ago.

Reply:
[black](https://github.com/psf/black) is a popular (18.5K stars) _PEP-8_ compliant formatter that prevents [bikeshedding](https://en.wikipedia.org/wiki/Law_of_triviality) around code styling, just like `rustfmt`, `gofmt`, `ktlint`, `google-java-format` etc.

By using such tools, one can eliminate the human factor leading to a mix in style across the codebase. Instead, everything always follows the same style and is in-sync.

Reply:
It's good to use the same style. But I think it should only be added to tox if the usage is agreed in the project

Reply:
Enjoy the chaos!

Reply:
I'm not a project member and if a test fails because of the coding style I'll fix it.
I no each programer want to use it's own style and that's difficult to find an agreement.
It's on your side to deactivate the black test or to use the style from it. I had one place where I turned of the formatting because the if branch had a line break and the else branch not (see `# fmt: off` in `jira.py`:

Reply:
How should I proceed? I know each programmer has it's own style and it's sometimes difficult to read other code.
My opinion:
- Keep black
- Introduce a `pyproject.toml` for the configuration as mentioned in the documentation:
  - Move pylint and black configuration to this file 
  - Change the line length from the default (88) to 100 or 120.

Reply:
So let's use one util/linter. (if it's ok to use everywhere  - so it's black) Let's do it. 
only one suggestion, sometimes so hard to read the method.